heiko/users.py

```diff
@@ -127,6 +127,34 @@ def create_user(auth, client):
         log("Error creating user", serv="ERROR")
         return False
 
+def transfer_coins(auth, client):
+    """
+    Transfer credits to another user
+
+    :auth_client: dict
+    :user_client: dict
+    :returns: bool
+    """
+
+    log("What user you want to send credits to?")
+    target_user = find_user_by_username(auth, client)
```

Comment:
Check return value of find_user_by_username. False if user wasn't found

Reply:
Nice addition, implemented